### PR TITLE
RAIL-3651 fix broken background while loading

### PR DESCRIPTION
**Pull request by plavemar**
- fix broken loading
- fix broken cypress tests

JIRA: RAIL-3651

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by plavemar** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5050/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6725/ : FAILURE in 11m 27s

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dateFilter.spec.ts**|7|-|15.7710s|
|✅|**dashboard.spec.ts**|9|-|16.9290s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/23/).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5051/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/23/ : SUCCESS in 11m 53s

File: libs/sdk-ui-tests-e2e/recordings/mappings/gdc_account_profile_b388eefffab41947076a75fd36d0bad1-78489080-dcbb-4a0b-bd02-a40cf2ca8985.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+        "body": "{\"accountSetting\":{\"ssoProvider\":null,\"updated\":\"2021-07-21 14:07:27\",\"timezone\":null,\"position\":null,\"phoneNumber\":\"720976661\",\"login\":\"martin.plavek@gooddata.com\",\"lastName\":\"Plavek\",\"firstName\":\"Martin\",\"created\":\"2021-04-07 11:36:41\",\"country\":null,\"companyName\":\"Good Data\",\"email\":\"martin.plavek@gooddata.com\",\"authenticationModes\":[],\"language\":\"en-US\",\"ipWhitelist\":null,\"effectiveIpWhitelist\":null,\"links\":{\"self\":\"/gdc/account/profile/b388eefffab41947076a75fd36d0bad1\",\"projects\":\"/gdc/account/profile/b388eefffab41947076a75fd36d0bad1/projects\",\"domain\":\"/gdc/domains/default\",\"auditEvents\":\"/gdc/account/profile/b388eefffab41947076a75fd36d0bad1/auditEvents\"}}}",
```
Comment (no23reason) [inline libs/sdk-ui-tests-e2e/recordings/mappings/gdc_account_profile_b388eefffab41947076a75fd36d0bad1-78489080-dcbb-4a0b-bd02-a40cf2ca8985.json:10]: Delete your phone number.

**Comment by lupko** [comment]
@plavemar please also drop the jira ticket prefix from commits; instead have RELATED: RAIL-#### in the commit desc

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5052/ : SUCCESS in 50s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6726/ : SUCCESS in 30m 23s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1701

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1435/ : FAILURE in 2m 55s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1712/ : SUCCESS in 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/497/ : SUCCESS in 15m 24s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6730/ : SUCCESS in 23m 34s

**Pull request merged**